UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c

```diff
@@ -1039,6 +1032,26 @@ PrintSolStrings (
   }
 }
 
+/**
+  Refresh the logo on ReadyToBoto event. It will clear the screen from strings
+  and progress bar when timeout is reached or continue key is pressed.
+
+  @param    Event          Event pointer.
+  @param    Context        Context pass to this function.
+**/
+VOID
+EFIAPI
+RefreshLogo (
+  IN EFI_EVENT    Event,
+  IN VOID         *Context
+  )
+{
+  gBS->CloseEvent (Event);
+  gBS->Stall (100 * 1000);

```

Comment:
Can this be moved after screen is redrawn?

Why is this delay even needed in the first place? Just to show 100% on progress bar for a while?

Reply:
>  Just to show 100% on progress bar for a while?

Sort of... We may get rid of that I think

Reply:
```suggestion
```